source/renderer/app/components/hardware-wallet/settings/ConnectHardwareWallet.js

```diff
@@ -0,0 +1,154 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, FormattedHTMLMessage, intlShape } from 'react-intl';
+import SVGInline from 'react-svg-inline';
+import ledgerIcon from '../../../assets/images/hardware-wallet/ledger-cropped.inline.svg';
+import ledgerXIcon from '../../../assets/images/hardware-wallet/ledger-x-cropped-outlines.inline.svg';
+import trezorIcon from '../../../assets/images/hardware-wallet/trezor-ledger.inline.svg';
+import exportIcon from '../../../assets/images/hardware-wallet/export.inline.svg';
+import checkIcon from '../../../assets/images/hardware-wallet/check.inline.svg';
+import clearIcon from '../../../assets/images/hardware-wallet/close-cross-red.inline.svg';
+import ledgerSmallIcon from '../../../assets/images/hardware-wallet/ledger-bold-ic.inline.svg';
+import styles from './ConnectHardwareWallet.scss';
+import LoadingSpinner from '../../widgets/LoadingSpinner';
+
+const messages = defineMessages({
+  hardwareWalletTitle: {
+    id: 'wallet.hardware.hardwareWalletTitle',
+    defaultMessage: '!!!Hardware wallet',
+    description: 'Hardware wallet title.',
+  },
+  ledgerWalletTitle: {
+    id: 'wallet.hardware.ledgerWalletTitle',
+    defaultMessage: '!!!Ledger wallet',
+    description: 'Ledger wallet title.',
+  },
+  hardwareWalletInstructions: {
+    id: 'wallet.hardware.hardwareWalletInstructions',
+    defaultMessage: '!!!Follow instructions to access your wallet',
+    description: 'Follow instructions label',
+  },
+  hardwareWalletLedgerBegin: {
+    id: 'wallet.hardware.hardwareWalletLedgerBegin',
+    defaultMessage:
+      '!!!To begin, connect and unlock your <span>Ledger Device</span>',
+    description: 'Connect device label',
+  },
+  hardwareWalletBegin: {
+    id: 'wallet.hardware.hardwareWalletBegin',
+    defaultMessage:
+      '!!!To begin, connect and unlock your <span>Hardware wallet Device</span>',
+    description: 'Connect device label',
+  },
+  hardwareWalletExport: {
+    id: 'wallet.hardware.hardwareWalletExport',
+    defaultMessage: '!!!Export <span>public key</span> on your device',
+    description: 'Export wallet label',
+  },
+});
+
+type Props = {
+  onOpenExternalLink: Function,
+  isLedger: boolean,
+  isTrezor: boolean,
+  isDeviceConnected: boolean | null,
+  fetchingDevice: boolean,
+  exportingExtendedPublicKey: boolean | null,
+  isExportingPublicKeyAborted: boolean,
+};
+
+@observer
+export default class ConnectHardwareWallet extends Component<Props> {
+  static contextTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  render() {
+    const { intl } = this.context;
+
+    const link = '';
```

Comment:
@DeeJayElly add a real translated link. Maybe we don't have a page for that but you can redirect to https://daedaluswallet.io/  || https://daedaluswallet.io/ja  maybe for now?

Reply:
Added